tsl/test/expected/bgw_policy.out

```diff
@@ -420,14 +420,16 @@ select check_chunk_oid(:oldest_chunk_id, :reorder_chunk_oid);
 -- Should be noop again
 select * from test_reorder(:reorder_job_id) \gset  reorder_
 NOTICE:  no chunks need reordering for hypertable public.test_table
-CREATE TABLE test_table_int(time int);
+CREATE TABLE test_table_int(time bigint, junk int);
 SELECT create_hypertable('test_table_int', 'time', chunk_time_interval => 1);
 NOTICE:  adding not-null constraint to column "time"
       create_hypertable      
 -----------------------------
  (2,public,test_table_int,t)
 (1 row)
 
+create or replace function dummy_now() returns BIGINT LANGUAGE SQL IMMUTABLE as  'SELECT 1::BIGINT';
```

Comment:
So we probably need a test where you have funcs, 1) called dummy_now() which is fine, and another, called dummy_now(int) which is not fine, right now, you will get errors, and this is why we probably shouldn't take in a regproc...

Reply:
not sure what you mean by 'this is why we probably shouldn't take in a regproc...' ?
I thought as long as we catch the error with an informative message, it is ok?